test/main.cpp

```diff
@@ -324,6 +325,30 @@ BOOST_AUTO_TEST_CASE(Throwt) {
       },
       std::system_error
   );
+
+  try {
+    throw std::runtime_error("foobar");
+  } catch (const std::exception& e) {
+    std::stringstream ss;
+    yk::printt(ss, e);
+    BOOST_TEST(ss.str() == "foobar\n");
+  }
+
+  struct my_exception : std::runtime_error, std::nested_exception {
```

Comment:
Directly inheriting from `std::nested_exception` seems odd, you can just use [std::throw_with_nested](http://en.cppreference.com/w/cpp/error/throw_with_nested)